src/lib/modules/jitter/jitter.go

```diff
@@ -9,7 +9,16 @@ import (
 func HandleJitter(url *string, packets int) (jitter *float64) {
 
 	chnl := make(chan *string)
-	go utils.CLIPing(url, packets, chnl)
+	errChnl := make(chan error)
+	go utils.CLIPing(url, packets, chnl, errChnl)
+	errCode <- errChnl
 	resp := <-chnl
+	if errCode != nil {
+		log.Println(*resp)
+		close(errChnl)
+		close(chnl)
+		*jitter = 0
+		return
```

Comment:
could you make a second return param as an error, which would be returned in case of abnormal behaviour. This is becoz setting jitter to 0 will indicate an ideal system than any error. 
We can return the error and log it rather than panic. That should be the case.

What do you think?

Reply:
Yes. You are right. The error should be propagated back to caller function. I will make that change.
I thought of doing this in the same PR, but I wanted to hear your comments first. will make the change.